python/fledge/services/core/api/common.py

```diff
@@ -85,6 +86,12 @@ def services_health_litmus_test():
 
     status_color = services_health_litmus_test()
     safe_mode = True if server.Server.running_in_safe_mode else False
+    version = ""
+    with open(_FLEDGE_ROOT + '/VERSION') as f:
+        for line in f:
+            if 'fledge_version=' in line:
+                key, value = line.strip().split('=')
+                version = value
```

Comment:
let's move this to a **cached** def; so that we read only once on each start; 

Ping is already an expensive call; this loop + read file will make it more expensive. I can understand, we want to be sure to read the line having `fledge_version=` tough as we keep it always first line, may be we better use `.readline().split('=')[1].strip()`

Reply:
Given ping is used frequently by the GUI it is important we make it as low impact as possible. Let's read the version from the file once at startup, we canteen add the version in the startup log message.

Reply:
Yes, This is memoized now with https://github.com/fledge-iot/fledge/pull/243/commits/a77564be28c1a1f3219b3deade418596ee5a64b3